apps/communications/contacts/js/views/details.js

```diff
@@ -83,15 +84,18 @@ contacts.Details = (function() {
 
     initPullEffect(cover);
 
-    utils.listeners.add({
-      '#toggle-favorite': toggleFavorite,
-      '#details-view-header': [
-        {
-          event: 'action',
-          handler: handleDetailsBack
-        }
-      ],
-      '#edit-contact-button': showEditContact
+    // to avoid race conditions with NFC, we load it before handleDetails
+    LazyLoader.load('/contacts/js/nfc.js', () => {
+      utils.listeners.add({
```

Comment:
I had the feeling that we were removing the `utils.listeners`, @borjasalguero is that right?